include/RAJA/internal/type_traits.hpp

```diff
@@ -128,20 +128,20 @@ struct is_enum_same : std::false_type {
 template <typename T, T A>
 struct is_enum_same<T, A, A> : std::true_type {
 };
-template <typename Policy, Policy P>
+template <typename Policy, RAJA::Policy P>
 struct models_policy
     : std::integral_constant<bool,
-                             is_enum_same<Policy, Policy::policy, P>::value> {
+                             is_enum_same<RAJA::Policy, Policy::policy, P>::value> {
```

Comment:
I ended up needing to rename it anyways -- the compiler didn't like when I was overly specific